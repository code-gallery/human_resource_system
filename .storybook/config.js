import { configure } from '@storybook/react'

const loadStories = () => {
  require('../src/stories')
}

configure(loadStories, module)
