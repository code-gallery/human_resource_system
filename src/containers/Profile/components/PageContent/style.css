.PageContent .profile-experience-block .Avatar {
  float: left;
}

@media (max-width: 768px) {
  .PageContent .profile-experience-wrapper {
    padding: 10px;
  }

  .PageContent .profile-experience-block .Avatar {
    padding-bottom: 38px;
  }

  .PageContent .profile-experience-block .Avatar img {
    width: 104px;
    height: 104px;
    margin-right: 10px;
  }

  .PageContent .visible-xs .profile-status-ico {
    position: absolute;
    top: 114px;
    width: 30px;
    height: 30px;
    background-size: 22px 22px;
    left: 44px;
  }
}
