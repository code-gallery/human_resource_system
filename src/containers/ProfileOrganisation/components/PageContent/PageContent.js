import React, { Component } from 'react'
import PropTypes from 'prop-types'
import { Link } from 'react-router-dom'
import { ROUTE_URL } from 'containers/constants'
import AboutEdit from '../AboutEdit'
import SpecialitiesEdit from '../SpecialitiesEdit'
import DetailsEdit from '../DetailsEdit'
import About from '../About'
import Employees from '../Employees'
import Students from '../Students'
import TagList from '../TagList'

class PageContent extends Component {
  renderEditMode() {
    const { organisation, reference, onChangeInput, errors } = this.props
    let accrediationLink = ROUTE_URL.orgAccreditations
    accrediationLink = accrediationLink.replace(':orgId', organisation.id)
    let verificationsLink = ROUTE_URL.orgVerifications
    verificationsLink = verificationsLink.replace(':orgId', organisation.id)
    let adminsLink = ROUTE_URL.orgAdmins
    adminsLink = adminsLink.replace(':orgId', organisation.id)

    return (
      <div className="container ProfileOrganisation-Edit">
        <div className="row">
          <div className="col-xs-12 col-md-8">
            <AboutEdit
              organisation={organisation}
              onChangeInput={onChangeInput}
              errors={errors}
            />
            <div className="hidden-md hidden-lg">
              <DetailsEdit
                organisation={organisation}
                reference={reference}
                onChangeInput={onChangeInput}
                errors={errors}
              />
              <SpecialitiesEdit
                organisation={organisation}
                onChangeInput={onChangeInput}
                errors={errors}
              />
            </div>
            <Employees
              editMode={true}
              organisation={organisation}
            />
            <Students
              editMode={true}
              organisation={organisation}
            />
          </div>
          <div className="aside col-md-4 hidden-sm hidden-xs">
            <Link to={verificationsLink} className="btn blue-btn">View Verifications</Link>
            <Link to={accrediationLink} className="btn blue-btn">Accreditations Admin</Link>
            <Link to={adminsLink} className="btn blue-btn">Manage Contacts</Link>
            <DetailsEdit
              organisation={organisation}
              reference={reference}
              onChangeInput={onChangeInput}
              errors={errors}
            />
            <SpecialitiesEdit
              organisation={organisation}
              onChangeInput={onChangeInput}
              errors={errors}
            />
          </div>
        </div>
      </div>
    )
  }

  renderReadOnly() {
    const { organisation } = this.props
    const { specialities } = organisation
    const tags = (specialities) ? specialities.split(',') : []

    return (
      <div className="container">
        <div className="row">
          <div className="col-xs-12 col-md-8">
            <About organisation={organisation} />
            <Employees organisation={organisation} />
            <Students organisation={organisation} />
          </div>
          <div className="col-md-4 hidden-sm hidden-xs">
            <TagList tags={tags} />
          </div>
        </div>
      </div>
    )
  }

  render() {
    const { editMode } = this.props

    if (editMode) {
      return this.renderEditMode()
    }

    return this.renderReadOnly()
  }
}

PageContent.propTypes = {
  editMode: PropTypes.bool.isRequired,
  organisation: PropTypes.object,
  reference: PropTypes.shape({
    awards: PropTypes.array,
    degrees: PropTypes.array,
    industries: PropTypes.array,
    gapsReasons: PropTypes.array
  }),
  onChangeInput: PropTypes.func.isRequired,
  errors: PropTypes.array.isRequired
}

export default PageContent
