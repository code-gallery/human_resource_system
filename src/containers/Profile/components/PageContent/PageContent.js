import React, { Component } from 'react'
import PropTypes from 'prop-types'
import _keys from 'lodash/keys'
import DragSortableList from 'react-drag-sortable'
import { DEFAULT_ORDER } from 'containers/constants'
import EducationList from '../EducationList'
import WorkExperience from '../WorkExperience'
import AwardList from '../AwardList'
import RightBlock from '../OverviewBlock'
import EditProfile from '../OverviewBlock/components/EditProfile'
import './style.css'

function defaultOrder(pageOrder) {
  if (Object.keys(pageOrder).length < 9) {
    return DEFAULT_ORDER
  } else {
    return pageOrder
  }
}

class PageContent extends Component {
  constructor(props) {
    super(props)
    const pageOrder = props.pageOrder ? props.pageOrder : {}
    this.state = {
      reorder: false,
      order: defaultOrder(pageOrder)
    }
  }

  componentWillReceiveProps(nextProps) {
    if (this.props.pageOrder !== nextProps.pageOrder &&
      _keys(nextProps.pageOrder).length !== 0) {
      this.setState({
        order: nextProps.pageOrder
      })
    }
    if (this.props.editMode !== nextProps.editMode) {
      this.setState({
        reorder: false
      })
    }
  }

  toggleReorder = () => {
    this.setState({
      reorder: !this.state.reorder
    })
  }

  onSort = (sortedList) => {
    const _obj = {}
    sortedList.map((item) => {
      _obj[item.classes[0]] = item.rank
      return true
    })
    this.props.onChangeProfileField('profile_order', _obj)
    return true
  }

  renderComponents() {
    const { reorder, order } = this.state
    const {
      editMode, educations, jobs,gaps, reference,
      deleteWork, deleteEducation, deleteAward
    } = this.props
    const { degrees, awards, industries, highSchoolDegrees, gapsReasons } = reference
    const {
      user,
      allAwards: {
        award,
        cpd,
        language,
        certificate,
        achievement,
        project,
        skill
      }
    } = this.props
    const elements = {
      education: (<EducationList
        education={educations}
        editMode={editMode}
        awards={awards}
        highSchoolDegrees={highSchoolDegrees}
        qualifications={degrees}
        reorder={reorder}
        toggleReorder={this.toggleReorder}
        deleteEducation={deleteEducation}
        saveEntity={this.props.saveEntity}
        filterEntries={this.props.filterEntries}
        user={user}
      />),
      jobs: (<WorkExperience
        jobs={jobs}
        editMode={editMode}
        industries={industries}
        gapsReasons={gapsReasons}
        reorder={reorder}
        deleteWork={deleteWork}
        toggleReorder={this.toggleReorder}
        saveEntity={this.props.saveEntity}
        filterEntries={this.props.filterEntries}
        user={user}
      />),
      award: (<AwardList
        awardType="award"
        awardTitle="Awards"
        awardList={award}
        editMode={editMode}
        reorder={reorder}
        toggleReorder={this.toggleReorder}
        deleteAward={deleteAward}
        saveAward={this.props.saveAward}
        filterEntries={this.props.filterEntries}
        user={user}
      />),
      cpd: (<AwardList
        awardType="cpd"
        awardTitle="Professional Development (CPD)"
        awardList={cpd}
        editMode={editMode}
        reorder={reorder}
        toggleReorder={this.toggleReorder}
        deleteAward={deleteAward}
        saveAward={this.props.saveAward}
        filterEntries={this.props.filterEntries}
        user={user}
      />),
      certificate: (<AwardList
        awardType="certificate"
        awardTitle="Certificates"
        awardList={certificate}
        editMode={editMode}
        reorder={reorder}
        toggleReorder={this.toggleReorder}
        deleteAward={deleteAward}
        saveAward={this.props.saveAward}
        filterEntries={this.props.filterEntries}
        user={user}
      />),
      achievement: (<AwardList
        awardType="achievement"
        awardTitle="Achievements"
        awardList={achievement}
        editMode={editMode}
        reorder={reorder}
        toggleReorder={this.toggleReorder}
        deleteAward={deleteAward}
        saveAward={this.props.saveAward}
        filterEntries={this.props.filterEntries}
        user={user}
      />),
      project: (<AwardList
        awardType="project"
        awardTitle="Projects"
        awardList={project}
        editMode={editMode}
        reorder={reorder}
        toggleReorder={this.toggleReorder}
        deleteAward={deleteAward}
        saveAward={this.props.saveAward}
        filterEntries={this.props.filterEntries}
        user={user}
      />),
      skill: (<AwardList
        awardType="skill"
        awardTitle="Skills"
        awardList={skill}
        editMode={editMode}
        reorder={reorder}
        toggleReorder={this.toggleReorder}
        deleteAward={deleteAward}
        saveAward={this.props.saveAward}
        filterEntries={this.props.filterEntries}
        user={user}
      />),
      language: (<AwardList
        awardType="language"
        awardTitle="Languages"
        awardList={language}
        editMode={editMode}
        reorder={reorder}
        toggleReorder={this.toggleReorder}
        deleteAward={deleteAward}
        saveAward={this.props.saveAward}
        filterEntries={this.props.filterEntries}
        user={user}
      />)
    }
    const _order = Object.keys(order).sort(function (a, b) {
      return order[a] - order[b]
    })
    const list = _order.map((key) => {
      return { content: (elements[key]), classes: [ key ] }
    })

    return (
      <DragSortableList
        items={list}
        moveTransitionDuration={1}
        dropBackTransitionDuration={0.3}
        onSort={this.onSort}
        type="vertical"
        disableDrag={!(reorder && editMode)}
      />
    )
  }

  render() {
    const { onChangeProfileField, allAwards, jobs,  educations, editMode, user, activities } = this.props
    const invalidFields = []
    if (user.first_name === '') {
      invalidFields.push('first_name')
    }
    if (user.last_name === '') {
      invalidFields.push('last_name')
    }

    return (
      <div className="container PageContent">
        <div className="row">
          {editMode &&
            <div className="col-xs-12 visible-xs visible-sm">
              <EditProfile
                onChangeProfileField={onChangeProfileField}
                user={user}
                invalidFields={invalidFields}
              />
            </div>
          }
          <div className="col-xs-12 col-md-8">
            {this.renderComponents()}
          </div>
          <RightBlock
            allAwards={allAwards}
            jobs={jobs}
            educations={educations}
            editMode={editMode}
            user={user}
            onChangeProfileField={onChangeProfileField}
            activities={activities}
          />
        </div>
      </div>
    )
  }
}

PageContent.propTypes = {
  pageOrder: PropTypes.object,
  user: PropTypes.object,
  editMode: PropTypes.bool,
  allAwards: PropTypes.object,
  educations: PropTypes.array,
  activities: PropTypes.array,
  jobs: PropTypes.array,
  reference: PropTypes.shape({
    awards: PropTypes.array,
    degrees: PropTypes.array,
    industries: PropTypes.array,
    gapsReasons: PropTypes.array
  }),
  onChangeProfileField: PropTypes.func.isRequired,
  deleteAward: PropTypes.func.isRequired,
  deleteWork: PropTypes.func.isRequired,
  deleteEducation: PropTypes.func.isRequired,
  saveAward: PropTypes.func.isRequired,
  saveEntity: PropTypes.func.isRequired,
  filterEntries: PropTypes.func.isRequired
}

export default PageContent
