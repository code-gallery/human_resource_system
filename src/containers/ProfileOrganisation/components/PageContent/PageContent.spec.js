import React from 'react'
import renderer from 'react-test-renderer'
import { Provider } from 'react-redux'
import { BrowserRouter as Router } from 'react-router-dom'
import config from 'store'
import PageContent from './index'

describe('containers/ProfileOrganisation/components/PageContent', () => {
  const { store } = config
  const commonProps = {
    organisation: {
      id: 12,
      name: 'Applied Blockchain',
      specialities: 'UX,JS,UI',
      employees: [
        { unique_key: 'jane', profile_image: null, first_name: 'jane', last_name: 'doe', tagline: 'hello' },
        { id: '21', profile_image: 'https://profile_image', first_name: 'john', last_name: 'doe', tagline: 'hello' }
      ],
      students: []
    },
    reference: {
      industries: [
        { text: 'Accounting' },
        { text: 'Banking' },
        { text: 'IT' }
      ],
      organisationSize: [
        { text: '1-10 employees' },
        { text: '11-50 employees' },
        { text: '51-200 employees' }
      ]
    },
    errors: [],
    onChangeInput: jest.fn()
  }

  describe('when editMode is false', () => {
    it('renders correctly', () => {
      const props = {
        ...commonProps,
        editMode: false
      }
      const tree = renderer.create(
        <Provider store={store}>
          <Router>
            <PageContent {...props} />
          </Router>
        </Provider>
      ).toJSON()

      expect(tree).toMatchSnapshot()
    })
  })

  describe('when editMode is true', () => {
    it('renders correctly', () => {
      const props = {
        ...commonProps,
        editMode: true
      }
      const tree = renderer.create(
        <Provider store={store}>
          <Router>
            <PageContent {...props} />
          </Router>
        </Provider>
      ).toJSON()

      expect(tree).toMatchSnapshot()
    })
  })
})
