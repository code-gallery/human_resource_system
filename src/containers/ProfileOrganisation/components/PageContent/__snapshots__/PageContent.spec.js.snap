// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`containers/ProfileOrganisation/components/PageContent when editMode is false renders correctly 1`] = `
<div
  className="container"
>
  <div
    className="row"
  >
    <div
      className="col-xs-12 col-md-8"
    >
      <div
        className="ContentCard col-xs-12 "
      >
        <div
          className="ContentCard-wrapper"
        >
          <h3
            className="ContentCard-title"
          >
            About
          </h3>
        </div>
      </div>
      <div
        className="ContentCard col-xs-12 "
      >
        <div
          className="ContentCard-wrapper"
        >
          <h3
            className="ContentCard-title"
          >
            Verified Employees
          </h3>
          <div
            className="row"
          >
            <div
              className="col-xs-6 col-sm-4"
            >
              <div
                className="AvatarBig"
              >
                <img
                  alt="jane doe"
                  src="appii-placeholder.png"
                />
                <p>
                  <a
                    href="/profile/jane"
                    onClick={[Function]}
                    title="jane doe"
                  >
                    jane doe
                  </a>
                  <span
                    title="hello"
                  >
                    hello
                  </span>
                </p>
              </div>
            </div>
            <div
              className="col-xs-6 col-sm-4"
            >
              <div
                className="AvatarBig"
              >
                <img
                  alt="john doe"
                  src="https://profile_image"
                />
                <p>
                  <a
                    href="/profile/21"
                    onClick={[Function]}
                    title="john doe"
                  >
                    john doe
                  </a>
                  <span
                    title="hello"
                  >
                    hello
                  </span>
                </p>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      className="col-md-4 hidden-sm hidden-xs"
    >
      <ul
        className="TagList"
      >
        <li>
          UX
        </li>
        <li>
          JS
        </li>
        <li>
          UI
        </li>
      </ul>
    </div>
  </div>
</div>
`;

exports[`containers/ProfileOrganisation/components/PageContent when editMode is true renders correctly 1`] = `
<div
  className="container ProfileOrganisation-Edit"
>
  <div
    className="row"
  >
    <div
      className="col-xs-12 col-md-8"
    >
      <div
        className="ContentCard col-xs-12 "
      >
        <div
          className="ContentCard-wrapper"
        >
          <h3
            className="ContentCard-title"
          >
            About
          </h3>
          <form>
            <div
              className="form-group"
            >
              <textarea
                className="edit-text-area"
                defaultValue={undefined}
                name="about_us"
                onChange={[Function]}
              />
            </div>
          </form>
        </div>
      </div>
      <div
        className="hidden-md hidden-lg"
      >
        <div
          className="ContentCard col-xs-12 DetailsEdit"
        >
          <div
            className="ContentCard-wrapper"
          >
            <h3
              className="ContentCard-title"
            >
              Details
            </h3>
            <form>
              <div
                className="form-group "
              >
                <label>
                  Name
                </label>
                <input
                  name="name"
                  onChange={[Function]}
                  placeholder="Company name"
                  required={false}
                  type="text"
                  value="Applied Blockchain"
                />
              </div>
              <div
                className="form-group"
              >
                <label>
                  Size
                </label>
                <div
                  className="Select is-clearable is-searchable Select--single"
                  style={undefined}
                >
                  <div
                    className="Select-control"
                    onKeyDown={[Function]}
                    onMouseDown={[Function]}
                    onTouchEnd={[Function]}
                    onTouchMove={[Function]}
                    onTouchStart={[Function]}
                    style={undefined}
                  >
                    <span
                      className="Select-multi-value-wrapper"
                      id="react-select-2--value"
                    >
                      <div
                        className="Select-placeholder"
                      >
                        Select...
                      </div>
                      <div
                        className="Select-input"
                        style={
                          Object {
                            "display": "inline-block",
                          }
                        }
                      >
                        <input
                          aria-activedescendant="react-select-2--value"
                          aria-describedby={undefined}
                          aria-expanded="false"
                          aria-haspopup="false"
                          aria-label={undefined}
                          aria-labelledby={undefined}
                          aria-owns=""
                          className={undefined}
                          id={undefined}
                          onBlur={[Function]}
                          onChange={[Function]}
                          onFocus={[Function]}
                          required={false}
                          role="combobox"
                          style={
                            Object {
                              "boxSizing": "content-box",
                              "width": "5px",
                            }
                          }
                          tabIndex={undefined}
                          value=""
                        />
                        <div
                          style={
                            Object {
                              "height": 0,
                              "left": 0,
                              "overflow": "scroll",
                              "position": "absolute",
                              "top": 0,
                              "visibility": "hidden",
                              "whiteSpace": "pre",
                            }
                          }
                        >
                          
                        </div>
                      </div>
                    </span>
                    <span
                      className="Select-arrow-zone"
                      onMouseDown={[Function]}
                    >
                      <span
                        className="Select-arrow"
                        onMouseDown={[Function]}
                      />
                    </span>
                  </div>
                </div>
              </div>
              <div
                className="form-group"
              >
                <label>
                  Location
                </label>
                <div>
                  <input
                    placeholder="Enter your address"
                  />
                </div>
              </div>
              <div
                className="form-group"
              >
                <label>
                  Industry
                </label>
                <div
                  className="Select is-clearable is-searchable Select--single"
                  style={undefined}
                >
                  <div
                    className="Select-control"
                    onKeyDown={[Function]}
                    onMouseDown={[Function]}
                    onTouchEnd={[Function]}
                    onTouchMove={[Function]}
                    onTouchStart={[Function]}
                    style={undefined}
                  >
                    <span
                      className="Select-multi-value-wrapper"
                      id="react-select-3--value"
                    >
                      <div
                        className="Select-placeholder"
                      >
                        Select...
                      </div>
                      <div
                        className="Select-input"
                        style={
                          Object {
                            "display": "inline-block",
                          }
                        }
                      >
                        <input
                          aria-activedescendant="react-select-3--value"
                          aria-describedby={undefined}
                          aria-expanded="false"
                          aria-haspopup="false"
                          aria-label={undefined}
                          aria-labelledby={undefined}
                          aria-owns=""
                          className={undefined}
                          id={undefined}
                          onBlur={[Function]}
                          onChange={[Function]}
                          onFocus={[Function]}
                          required={false}
                          role="combobox"
                          style={
                            Object {
                              "boxSizing": "content-box",
                              "width": "5px",
                            }
                          }
                          tabIndex={undefined}
                          value=""
                        />
                        <div
                          style={
                            Object {
                              "height": 0,
                              "left": 0,
                              "overflow": "scroll",
                              "position": "absolute",
                              "top": 0,
                              "visibility": "hidden",
                              "whiteSpace": "pre",
                            }
                          }
                        >
                          
                        </div>
                      </div>
                    </span>
                    <span
                      className="Select-arrow-zone"
                      onMouseDown={[Function]}
                    >
                      <span
                        className="Select-arrow"
                        onMouseDown={[Function]}
                      />
                    </span>
                  </div>
                </div>
              </div>
              <div
                className="form-group "
              >
                <label>
                  Year Founded
                </label>
                <input
                  name="year_founded"
                  onChange={[Function]}
                  placeholder="YYYY"
                  required={false}
                  type="text"
                  value=""
                />
              </div>
            </form>
          </div>
        </div>
        <div
          className="ContentCard col-xs-12 SpecialitiesEdit"
        >
          <div
            className="ContentCard-wrapper"
          >
            <h3
              className="ContentCard-title"
            >
              Specialities
            </h3>
            <ul
              className="TagList"
            >
              <li
                onClick={[Function]}
              >
                ux
                <span
                  className="TagList-remove"
                >
                  x
                </span>
              </li>
              <li
                onClick={[Function]}
              >
                js
                <span
                  className="TagList-remove"
                >
                  x
                </span>
              </li>
              <li
                onClick={[Function]}
              >
                ui
                <span
                  className="TagList-remove"
                >
                  x
                </span>
              </li>
            </ul>
            <form>
              <div
                className="form-group"
              >
                <input
                  onChange={[Function]}
                  placeholder="Type to add specialities"
                  type="text"
                  value=""
                />
                <button
                  className="btn blue-btn action-btn"
                  onClick={[Function]}
                >
                  Add
                </button>
              </div>
            </form>
          </div>
        </div>
      </div>
      <div
        className="ContentCard col-xs-12 edit-mode"
      >
        <div
          className="ContentCard-wrapper"
        >
          <h3
            className="ContentCard-title"
          >
            Verified Employees
          </h3>
          <div
            className="row"
          >
            <div
              className="col-xs-6 col-sm-4"
            >
              <div
                className="AvatarBig"
              >
                <img
                  alt="jane doe"
                  src="appii-placeholder.png"
                />
                <p>
                  <a
                    href="/profile/jane"
                    onClick={[Function]}
                    title="jane doe"
                  >
                    jane doe
                  </a>
                  <span
                    title="hello"
                  >
                    hello
                  </span>
                </p>
              </div>
            </div>
            <div
              className="col-xs-6 col-sm-4"
            >
              <div
                className="AvatarBig"
              >
                <img
                  alt="john doe"
                  src="https://profile_image"
                />
                <p>
                  <a
                    href="/profile/21"
                    onClick={[Function]}
                    title="john doe"
                  >
                    john doe
                  </a>
                  <span
                    title="hello"
                  >
                    hello
                  </span>
                </p>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      className="aside col-md-4 hidden-sm hidden-xs"
    >
      <a
        className="btn blue-btn"
        href="/organisations/12/verifications"
        onClick={[Function]}
      >
        View Verifications
      </a>
      <a
        className="btn blue-btn"
        href="/organisations/12/accreditations"
        onClick={[Function]}
      >
        Accreditations Admin
      </a>
      <a
        className="btn blue-btn"
        href="/organisations/12/admins"
        onClick={[Function]}
      >
        Manage Contacts
      </a>
      <div
        className="ContentCard col-xs-12 DetailsEdit"
      >
        <div
          className="ContentCard-wrapper"
        >
          <h3
            className="ContentCard-title"
          >
            Details
          </h3>
          <form>
            <div
              className="form-group "
            >
              <label>
                Name
              </label>
              <input
                name="name"
                onChange={[Function]}
                placeholder="Company name"
                required={false}
                type="text"
                value="Applied Blockchain"
              />
            </div>
            <div
              className="form-group"
            >
              <label>
                Size
              </label>
              <div
                className="Select is-clearable is-searchable Select--single"
                style={undefined}
              >
                <div
                  className="Select-control"
                  onKeyDown={[Function]}
                  onMouseDown={[Function]}
                  onTouchEnd={[Function]}
                  onTouchMove={[Function]}
                  onTouchStart={[Function]}
                  style={undefined}
                >
                  <span
                    className="Select-multi-value-wrapper"
                    id="react-select-4--value"
                  >
                    <div
                      className="Select-placeholder"
                    >
                      Select...
                    </div>
                    <div
                      className="Select-input"
                      style={
                        Object {
                          "display": "inline-block",
                        }
                      }
                    >
                      <input
                        aria-activedescendant="react-select-4--value"
                        aria-describedby={undefined}
                        aria-expanded="false"
                        aria-haspopup="false"
                        aria-label={undefined}
                        aria-labelledby={undefined}
                        aria-owns=""
                        className={undefined}
                        id={undefined}
                        onBlur={[Function]}
                        onChange={[Function]}
                        onFocus={[Function]}
                        required={false}
                        role="combobox"
                        style={
                          Object {
                            "boxSizing": "content-box",
                            "width": "5px",
                          }
                        }
                        tabIndex={undefined}
                        value=""
                      />
                      <div
                        style={
                          Object {
                            "height": 0,
                            "left": 0,
                            "overflow": "scroll",
                            "position": "absolute",
                            "top": 0,
                            "visibility": "hidden",
                            "whiteSpace": "pre",
                          }
                        }
                      >
                        
                      </div>
                    </div>
                  </span>
                  <span
                    className="Select-arrow-zone"
                    onMouseDown={[Function]}
                  >
                    <span
                      className="Select-arrow"
                      onMouseDown={[Function]}
                    />
                  </span>
                </div>
              </div>
            </div>
            <div
              className="form-group"
            >
              <label>
                Location
              </label>
              <div>
                <input
                  placeholder="Enter your address"
                />
              </div>
            </div>
            <div
              className="form-group"
            >
              <label>
                Industry
              </label>
              <div
                className="Select is-clearable is-searchable Select--single"
                style={undefined}
              >
                <div
                  className="Select-control"
                  onKeyDown={[Function]}
                  onMouseDown={[Function]}
                  onTouchEnd={[Function]}
                  onTouchMove={[Function]}
                  onTouchStart={[Function]}
                  style={undefined}
                >
                  <span
                    className="Select-multi-value-wrapper"
                    id="react-select-5--value"
                  >
                    <div
                      className="Select-placeholder"
                    >
                      Select...
                    </div>
                    <div
                      className="Select-input"
                      style={
                        Object {
                          "display": "inline-block",
                        }
                      }
                    >
                      <input
                        aria-activedescendant="react-select-5--value"
                        aria-describedby={undefined}
                        aria-expanded="false"
                        aria-haspopup="false"
                        aria-label={undefined}
                        aria-labelledby={undefined}
                        aria-owns=""
                        className={undefined}
                        id={undefined}
                        onBlur={[Function]}
                        onChange={[Function]}
                        onFocus={[Function]}
                        required={false}
                        role="combobox"
                        style={
                          Object {
                            "boxSizing": "content-box",
                            "width": "5px",
                          }
                        }
                        tabIndex={undefined}
                        value=""
                      />
                      <div
                        style={
                          Object {
                            "height": 0,
                            "left": 0,
                            "overflow": "scroll",
                            "position": "absolute",
                            "top": 0,
                            "visibility": "hidden",
                            "whiteSpace": "pre",
                          }
                        }
                      >
                        
                      </div>
                    </div>
                  </span>
                  <span
                    className="Select-arrow-zone"
                    onMouseDown={[Function]}
                  >
                    <span
                      className="Select-arrow"
                      onMouseDown={[Function]}
                    />
                  </span>
                </div>
              </div>
            </div>
            <div
              className="form-group "
            >
              <label>
                Year Founded
              </label>
              <input
                name="year_founded"
                onChange={[Function]}
                placeholder="YYYY"
                required={false}
                type="text"
                value=""
              />
            </div>
          </form>
        </div>
      </div>
      <div
        className="ContentCard col-xs-12 SpecialitiesEdit"
      >
        <div
          className="ContentCard-wrapper"
        >
          <h3
            className="ContentCard-title"
          >
            Specialities
          </h3>
          <ul
            className="TagList"
          >
            <li
              onClick={[Function]}
            >
              ux
              <span
                className="TagList-remove"
              >
                x
              </span>
            </li>
            <li
              onClick={[Function]}
            >
              js
              <span
                className="TagList-remove"
              >
                x
              </span>
            </li>
            <li
              onClick={[Function]}
            >
              ui
              <span
                className="TagList-remove"
              >
                x
              </span>
            </li>
          </ul>
          <form>
            <div
              className="form-group"
            >
              <input
                onChange={[Function]}
                placeholder="Type to add specialities"
                type="text"
                value=""
              />
              <button
                className="btn blue-btn action-btn"
                onClick={[Function]}
              >
                Add
              </button>
            </div>
          </form>
        </div>
      </div>
    </div>
  </div>
</div>
`;
